JYH/1Day/짝수와 홀수.js

```diff
@@ -0,0 +1,11 @@
+function solution(num) {
+    let answer;
+    if(num % 2 === 0){
+        answer = "Even"
+    }
+    else{
+        answer = "Odd";
+    }
+    return answer;
+}
```

Comment:
else 는 굳이 줄바꿈 하지 않으셔도 됩니다 ㅎㅎ

```suggestion
function solution(num) {
    let answer;
    if (num % 2 === 0){
        answer = "Even"
    } else {
        answer = "Odd";
    }
    return answer;
}
```
또는 아래와 같이 바로 리턴을 해주는 방법도 있습니다
```ts
function solution(num) {
  if (num % 2 === 0){
    return "Even";
  }
  return "Odd";
}
```